### PR TITLE
feat: added profile page & responsiveness to video library

### PR DESCRIPTION
**Pull request by kedark152**
### feat: added profile page & responsiveness to video library

Preview Link: https://deploy-preview-8--agroplay.netlify.app/

1. Added My Profile Page & created route link. 
2. Added responsiveness to entire app

In Mobile Responsive Layout:
4. changed the position of sidebar to down.
5. added links of playlist, watch later history in profile page.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (Hamdrive) [inline src/App.css]: if this CSS file is not being used it can be removed

File: src/routes/PageRoutes.js [hunk 2]
```diff
@@ -53,8 +53,15 @@ export const PageRoutes = () => {
           </RequiresAuth>
         }
       />
+      <Route
+        path="/profile"
+        element={
+          <RequiresAuth>
+            <MyProfile />
+          </RequiresAuth>
```
Comment (Hamdrive) [inline src/routes/PageRoutes.js:61]: You can try to implemented nested conditional routes

**Review by Hamdrive — COMMENTED** [review]
Like the implementation of window width to define type of user device, and handle responsiveness accordingly
Good use of CL utility classes

**Comment by ajit-sharmaGH** [comment]
Good naming Convention

File: src/routes/PageRoutes.js (hunk 2 above)
Comment (kedark152) [reply src/routes/PageRoutes.js:61]: Thanks for Review, will implement it later.

File: src/App.css (hunk 1 above)
Comment (kedark152) [reply src/App.css]: Implemented!

**Comment by kedark152** [comment]
Thanks @Hamdrive & @ajit-sharmaGH  for PR Review.

**Pull request merged**